DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: bioRad
 Title: Biological Analysis and Visualization of Weather Radar Data
-Version: 0.7.0.9588
+Version: 0.7.0.95977
```

Comment:
My PR on Github got stuck due to what seemed a github bug (did not want to refresh latest commits, and therefore kept erroring out), so I added this as a commit to try to get it out of the stucked state, but then forgot to remove the extra 7.

Reply:
let's move to 9001, 9002, 9003 numbering when we start the new 0.8 dev version

Reply:
Ok, let's hope we remember :-)

Reply:
I added a reminder in the todo list here: https://github.com/adokter/bioRad/issues/599

Reply:
@peterdesmet  & @adokter  Quick note is it really useful to always keep changing the version number as both `packageDescription` and `devtools::session_info` nicely report the git hash used for installing when installed with `devtools::install_github` (see below). This would mean only ambiguity would arise about the exact version installed if you locally install using `devtools::install` (note it would still report that it was installed locally, and a local install means there is anyway ambiguity about the exact code installed).
``` r
require(bioRad)
#> Loading required package: bioRad
#> Warning in fun(libname, pkgname): Package 'vol2birdR' is not found. Please see
#> https://adokter.github.io/vol2birdR/ for installation instructions.
#> Welcome to bioRad version 0.7.0.95977
#> The legacy packages maptools, rgdal, and rgeos, underpinning this package
#> will retire shortly. Please refer to R-spatial evolution reports on
#> https://r-spatial.org/r/2023/05/15/evolution4.html for details.
#> This package is now running under evolution status 0
#> Assigning sp_evolution_status to 2. See sp::get_evolution_status()
#> This is required until the 'sp' package deprecates 'rgdal'
utils::packageDescription('bioRad')
#> Package: bioRad
#> Title: Biological Analysis and Visualization of Weather Radar Data
#> Version: 0.7.0.95977
#> Description: Extract, visualize and summarize aerial movements of birds
#>         and insects from weather radar data. See
#>         <doi:10.1111/ecog.04028> for a software paper describing
#>         package and methodologies.
#> Authors@R: c( person("Adriaan M.", "Dokter", role = c("aut", "cre"),
#>         email = "amd427@cornell.edu", comment = c(ORCID =
#>         "0000-0001-6573-066X")), person("Peter", "Desmet", role =
#>         "aut", email = "peter.desmet@inbo.be", comment = c(ORCID =
#>         "0000-0002-8442-8025")), person("Bart", "Hoekstra", role =
#>         "ctb", email = "b.d.hoekstra@uva.nl", comment = c(ORCID =
#>         "0000-0002-7085-3805")), person("Pieter", "Huybrechts", role =
#>         "ctb", email = "pieter.huybrechts@inbo.be", comment = c(ORCID =
#>         "0000-0002-6658-6062")), person("Bart", "Kranstauber", role =
#>         "ctb", email = "b.kranstauber@uva.nl", comment = c(ORCID =
#>         "0000-0001-8303-780X")), person("Hidde", "Leijnse", role =
#>         "ctb", email = "hidde.leijnse@knmi.nl", comment = c(ORCID =
#>         "0000-0001-7835-4480")), person("Cecilia", "Nilsson", role =
#>         "ctb", email = "cecilia.nilsson709@gmail.com", comment =
#>         c(ORCID = "0000-0001-8957-4411")), person("Nicolas", "Noé",
#>         role = "ctb", email = "nicolas@niconoe.eu", comment = c(ORCID =
#>         "0000-0002-9503-4750")), person("Raphaël", "Nussbaumer", role =
#>         "ctb", email = "rafnuss@gmail.com", comment = c(ORCID =
#>         "0000-0002-8185-1020")), person("Jurriaan", "Spaaks", role =
#>         "ctb", email = "j.spaaks@esciencecenter.nl"),
#>         person("Alexander", "Tedeschi", role = "ctb", email =
#>         "alexander.tedeschi@cornell.edu", comment = c(ORCID =
#>         "0000-0003-0772-6931")), person("Stijn", "Van Hoey", role =
#>         "aut", email = "stijnvanhoey@gmail.com", comment = c(ORCID =
#>         "0000-0001-6413-3185")), person("Lourens", "Veen", role =
#>         "ctb", email = "l.veen@esciencecenter.nl"), person("Liesbeth",
#>         "Verlinden", role = "ctb", email = "l.verlinden@uva.nl",
#>         comment = c(ORCID = "0000-0003-1744-9325")) )
#> License: MIT + file LICENSE
#> URL: https://github.com/adokter/bioRad/,
#>         https://adriaandokter.com/bioRad/
#> BugReports: https://github.com/adokter/bioRad/issues
#> biocViews:
#> Depends: R (>= 3.5.0)
#> Imports: assertthat, curl, fields, ggplot2, glue, graphics, lubridate,
#>         lutz, maptools, methods, raster, readr, dplyr (>= 1.1.0),
#>         rhdf5, sp, sf, stats, tidyr, utils, viridisLite, viridis, rlang
#> Remotes: bioc::rhdf5
#> Suggests: vol2birdR, aws.s3, ggmap (>= 3.0.0), knitr, tidyselect,
#>         rmarkdown, testthat (>= 3.0.0), vdiffr, rgdal
#> LazyData: true
#> Encoding: UTF-8
#> VignetteBuilder: knitr
#> RoxygenNote: 7.2.3
#> Roxygen: list(markdown = TRUE)
#> Config/testthat/edition: 3
#> RemoteType: github
#> RemoteHost: api.github.com
#> RemoteRepo: bioRad
#> RemoteUsername: adokter
#> RemoteRef: HEAD
#> RemoteSha: 918308645f9b18fec667c8343e14d4b95b58883c
#> GithubRepo: bioRad
#> GithubUsername: adokter
#> GithubRef: HEAD
#> GithubSHA1: 918308645f9b18fec667c8343e14d4b95b58883c
#> NeedsCompilation: no
#> Packaged: 2023-06-08 07:40:58 UTC; bart
#> Author: Adriaan M. Dokter [aut, cre]
#>         (<https://orcid.org/0000-0001-6573-066X>), Peter Desmet [aut]
#>         (<https://orcid.org/0000-0002-8442-8025>), Bart Hoekstra [ctb]
#>         (<https://orcid.org/0000-0002-7085-3805>), Pieter Huybrechts
#>         [ctb] (<https://orcid.org/0000-0002-6658-6062>), Bart
#>         Kranstauber [ctb] (<https://orcid.org/0000-0001-8303-780X>),
#>         Hidde Leijnse [ctb] (<https://orcid.org/0000-0001-7835-4480>),
#>         Cecilia Nilsson [ctb]
#>         (<https://orcid.org/0000-0001-8957-4411>), Nicolas Noé [ctb]
#>         (<https://orcid.org/0000-0002-9503-4750>), Raphaël Nussbaumer
#>         [ctb] (<https://orcid.org/0000-0002-8185-1020>), Jurriaan
#>         Spaaks [ctb], Alexander Tedeschi [ctb]
#>         (<https://orcid.org/0000-0003-0772-6931>), Stijn Van Hoey [aut]
#>         (<https://orcid.org/0000-0001-6413-3185>), Lourens Veen [ctb],
#>         Liesbeth Verlinden [ctb]
#>         (<https://orcid.org/0000-0003-1744-9325>)
#> Maintainer: Adriaan M. Dokter <amd427@cornell.edu>
#> Built: R 4.1.2; ; 2023-06-08 07:40:59 UTC; unix
#> 
#> -- File: /home/bart/R/x86_64-pc-linux-gnu-library/4.1/bioRad/Meta/package.rds
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.1.2 (2021-11-01)
#>  os       Ubuntu 22.04.2 LTS
#>  system   x86_64, linux-gnu
#>  ui       X11
#>  language (EN)
#>  collate  en_US.UTF-8
#>  ctype    en_US.UTF-8
#>  tz       Europe/Amsterdam
#>  date     2023-06-08
#>  pandoc   2.19.2 @ /usr/lib/rstudio/resources/app/bin/quarto/bin/tools/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version     date (UTC) lib source
#>  bioRad      * 0.7.0.95977 2023-06-08 [1] Github (adokter/bioRad@9183086)
#>  cachem        1.0.8       2023-05-01 [1] CRAN (R 4.1.2)
#>  callr         3.7.3       2022-11-02 [1] CRAN (R 4.1.2)
#>  cli           3.6.1       2023-03-23 [1] CRAN (R 4.1.2)
#>  crayon        1.5.2       2022-09-29 [1] CRAN (R 4.1.2)
#>  devtools      2.4.5       2022-10-11 [1] CRAN (R 4.1.2)
#>  digest        0.6.31      2022-12-11 [1] CRAN (R 4.1.2)
#>  dplyr         1.1.2       2023-04-20 [1] CRAN (R 4.1.2)
#>  ellipsis      0.3.2       2021-04-29 [1] CRAN (R 4.1.2)
#>  evaluate      0.21        2023-05-05 [1] CRAN (R 4.1.2)
#>  fansi         1.0.4       2023-01-22 [1] CRAN (R 4.1.2)
#>  fastmap       1.1.1       2023-02-24 [1] CRAN (R 4.1.2)
#>  fs            1.6.2       2023-04-25 [1] CRAN (R 4.1.2)
#>  generics      0.1.3       2022-07-05 [1] CRAN (R 4.1.2)
#>  glue          1.6.2       2022-02-24 [1] CRAN (R 4.1.2)
#>  htmltools     0.5.5       2023-03-23 [1] CRAN (R 4.1.2)
#>  htmlwidgets   1.6.2       2023-03-17 [1] CRAN (R 4.1.2)
#>  httpuv        1.6.11      2023-05-11 [1] CRAN (R 4.1.2)
#>  knitr         1.43        2023-05-25 [1] CRAN (R 4.1.2)
#>  later         1.3.1       2023-05-02 [1] CRAN (R 4.1.2)
#>  lattice       0.21-8      2023-04-05 [1] CRAN (R 4.1.2)
#>  lifecycle     1.0.3       2022-10-07 [1] CRAN (R 4.1.2)
#>  magrittr      2.0.3       2022-03-30 [1] CRAN (R 4.1.2)
#>  memoise       2.0.1       2021-11-26 [1] CRAN (R 4.1.2)
#>  mime          0.12        2021-09-28 [1] CRAN (R 4.1.2)
#>  miniUI        0.1.1.1     2018-05-18 [1] CRAN (R 4.1.2)
#>  pillar        1.9.0       2023-03-22 [1] CRAN (R 4.1.2)
#>  pkgbuild      1.4.0       2022-11-27 [1] CRAN (R 4.1.2)
#>  pkgconfig     2.0.3       2019-09-22 [1] CRAN (R 4.1.2)
#>  pkgload       1.3.2       2022-11-16 [1] CRAN (R 4.1.2)
#>  prettyunits   1.1.1       2020-01-24 [1] CRAN (R 4.1.2)
#>  processx      3.8.1       2023-04-18 [1] CRAN (R 4.1.2)
#>  profvis       0.3.8       2023-05-02 [1] CRAN (R 4.1.2)
#>  promises      1.2.0.1     2021-02-11 [1] CRAN (R 4.1.2)
#>  ps            1.7.5       2023-04-18 [1] CRAN (R 4.1.2)
#>  purrr         1.0.1       2023-01-10 [1] CRAN (R 4.1.2)
#>  R.cache       0.16.0      2022-07-21 [1] CRAN (R 4.1.2)
#>  R.methodsS3   1.8.2       2022-06-13 [1] CRAN (R 4.1.2)
#>  R.oo          1.25.0      2022-06-12 [1] CRAN (R 4.1.2)
#>  R.utils       2.12.2      2022-11-11 [1] CRAN (R 4.1.2)
#>  R6            2.5.1       2021-08-19 [1] CRAN (R 4.1.2)
#>  Rcpp          1.0.10      2023-01-22 [1] CRAN (R 4.1.2)
#>  remotes       2.4.2       2021-11-30 [1] CRAN (R 4.1.2)
#>  reprex        2.0.2       2022-08-17 [1] CRAN (R 4.1.2)
#>  rlang         1.1.1       2023-04-28 [1] CRAN (R 4.1.2)
#>  rmarkdown     2.22        2023-06-01 [1] CRAN (R 4.1.2)
#>  rstudioapi    0.14        2022-08-22 [1] CRAN (R 4.1.2)
#>  sessioninfo   1.2.2       2021-12-06 [1] CRAN (R 4.1.2)
#>  shiny         1.7.4       2022-12-15 [1] CRAN (R 4.1.2)
#>  sp            1.6-1       2023-05-31 [1] CRAN (R 4.1.2)
#>  stringi       1.7.12      2023-01-11 [1] CRAN (R 4.1.2)
#>  stringr       1.5.0       2022-12-02 [1] CRAN (R 4.1.2)
#>  styler        1.10.1      2023-06-05 [1] CRAN (R 4.1.2)
#>  tibble        3.2.1       2023-03-20 [1] CRAN (R 4.1.2)
#>  tidyselect    1.2.0       2022-10-10 [1] CRAN (R 4.1.2)
#>  urlchecker    1.0.1       2021-11-30 [1] CRAN (R 4.1.2)
#>  usethis       2.2.0       2023-06-06 [1] CRAN (R 4.1.2)
#>  utf8          1.2.3       2023-01-31 [1] CRAN (R 4.1.2)
#>  vctrs         0.6.2       2023-04-19 [1] CRAN (R 4.1.2)
#>  withr         2.5.0       2022-03-03 [1] CRAN (R 4.1.2)
#>  xfun          0.39        2023-04-20 [1] CRAN (R 4.1.2)
#>  xtable        1.8-4       2019-04-21 [1] CRAN (R 4.1.2)
#>  yaml          2.3.7       2023-01-23 [1] CRAN (R 4.1.2)
#> 
#>  [1] /home/bart/R/x86_64-pc-linux-gnu-library/4.1
#>  [2] /usr/local/lib/R/site-library
#>  [3] /usr/lib/R/site-library
#>  [4] /usr/lib/R/library
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

Reply:
@peterdesmet @bart1 will the dev site still update if we stop incrementing the dev version?

Reply:
Yes, the dev site should update for every commit pushed to `master`